quanda/utils/common.py

```diff
@@ -313,6 +311,30 @@ def process_targets(
     return targets
 
 
+def move_to_device(
```

Comment:
Maybe let's call it `move_ds_item_to_device` or something like that, more descriptive?